docs/pages/admin-guides/deploy-a-cluster/license.mdx

```diff
@@ -151,7 +151,8 @@ Teleport Web UI users will see an alert banner:
 
 ![License expired banner](../../../img/enterprise/license-expired.png)
 
-Attempts to authenticate to Teleport via Single Sign-On connectors will fail.
+Attempts to authenticate to Teleport via SAML/OIDC Single Sign-On connectors will fail
+for example as an Enterprise feature.
```

Comment:
This grammar looks incorrect to me.

How about something like: Enterprise features like Single Sign-On will fail.

Reply:
thanks @zmb3 pls see update